src/components/PersonItem/PersonItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  person: Person;
+  selectedPersonSlug: string;
+};
+
+export const PersonItem: React.FC<Props> = ({ person, selectedPersonSlug }) => {
+  const {
+    slug,
+    sex,
+    born,
+    died,
+    fatherName = '-',
+    motherName = '-',
```

Comment:
If you continue to do so `motherName || '-'` then you don't need to set the default value.

Reply:
it doesn't work, I forgot to remove this value